src/core/components/Tooltip.tsx

```diff
@@ -0,0 +1,44 @@
+import React, { useState, useRef, useEffect } from 'react';
+import styles from './Tooltip.module.css';
+
+interface TooltipProps {
+  content: string;
+  children: React.ReactNode;
+  topMargin: string;
```

Comment:
Since we are building this from scratch, it will be useful to be able to manually position the tooltip.